web/web/src/lib/utils/initial.js

```diff
@@ -308,5 +308,33 @@ export const providers = [
         description: 'Apache Hudi catalog uri config'
       }
     ]
+  },
+  {
+    label: 'OceanBase',
+    value: 'jdbc-oceanbase',
+    defaultProps: [
+      {
+        key: 'jdbc-driver',
+        value: '',
+        required: true,
+        description: 'e.g. com.mysql.jdbc.Driver or com.mysql.cj.jdbc.Driver'
```

Comment:
`com.oceanbase.jdbc.Driver` is also available for OceanBase.
```suggestion
        description: 'e.g. com.mysql.jdbc.Driver or com.mysql.cj.jdbc.Driver' or `com.oceanbase.jdbc.Driver`
```